1224-user_administration/1442-prohibit_deletion_of_last_institution_admin.feature

```diff
@@ -0,0 +1,7 @@
+    @1442
+    Scenario: Prohibit deletion of last Institution Admin
+        Given that an App Admin or Institution Admin opens User Administration
+        And they see only one current Institution Admin
+        Then they see that the Delete button on its name is disabled
+        And a mouse-over informs the user that the last Institution Administrator can’t be deleted
```

Comment:
mouse-over er fint på PC, men finnes ikke for mobil. 
```suggestion
        And they see Information that the last Institution Administrator can’t be deleted
```

Reply:
jau, men mister samtidig ikke nattesøvn om noen logger inn på mobil for å fjerne alle admins i en institusjon, men ikke får eksplisitt beskjed om hvorfor hen ikke kan slette den siste

Reply:
er generelt imot mouse-over egentlig 😛  hva med svaksynte som navigerer med tastatur da? skal de ikke få mulighet til å vite dette de da?

Reply:
jeg syns egentlig det er såpass uviktig info at det er helt greit. 
Alternativet er at vi viser det hele tiden (selv om det er irrelevant hver gang man går dit uten intensjon om å slette seg selv som admin, som trolig er aldri), eller at knappen ikke er disabled men man får feilmelding når man prøver å slette

Kan like godt fjerne hele knappen også for meg, hvis det er bedre? (ikke at det gir mer feedback, men :P )

Reply:
siden det er `uviktig info`, så er det jo unødvendig å spesifisere at det skal være en mouse-over også da kanskje? kanskje det går an å fjerne hele denne linja?